molgenis-data-elasticsearch/src/main/java/org/molgenis/data/elasticsearch/ElasticsearchRepositoryCollection.java

```diff
@@ -92,11 +92,12 @@ public void deleteEntityMeta(String entityName)
 	@Override
 	public void addAttribute(String entityName, AttributeMetaData attribute)
 	{
-		EntityMetaData entityMetaData = dataService.getEntityMetaData(entityName);
+		EntityMetaData entityMetaData = dataService.getMeta().getEntityMetaData(entityName);
```

Comment:
shouldn't the repository get updated with the new metadata? or does that happen later?